src/App.js

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
 import './App.css';
 import { Home, About, Contact, Navigation } from './components';
+import { Route } from 'react-router-dom';
 
 const App = () => (
   <div>
     <Navigation />
+    <Route exact path="/" component={Home} />
```

Comment:
good job learning and executing on the basics of routing (I love how you made your 'component={page name here}' that's really cool.